### ISSUE TITLE
Alpine Edge?

### ISSUE THREAD

**Issue by asencis** [openresty/docker-openresty#162]
Is there a way we can get this working on Alpine Edge?

**Comment by neomantra** [comment]
We do not build Alpine Edge packages for Docker Hub, but you could build your own like so:

```
git clone https://github.com/openresty/docker-openresty.git
cd docker-openresty
docker build -t openresty-alpine-edge --build-arg RESTY_IMAGE_TAG="edge" -f alpine/Dockerfile .
```

*BUT* this isn't working.  Right right now it fails with `curl: (6) Could not resolve host: ftp.pcre.org`

I think it is an Alpine issue?  The `curl` invocation works on Alpine 3.12 but not on Edge...

```
docker run -it --rm alpine:edge   
/ # apk add -q curl
/ # curl -s -fSL https://ftp.pcre.org/pub/pcre/pcre-8.44.tar.gz -o pcre-8.44.tar.gz
curl: (6) Could not resolve host: ftp.pcre.org
```

```
docker run -it --rm alpine:3.12                                                                                                       
/ # apk add -q curl
/ # curl -s -fSL https://ftp.pcre.org/pub/pcre/pcre-8.44.tar.gz -o pcre-8.44.tar.gz
/ # 
```

**Comment by neomantra** [comment]
I just tried that curl test again and it is still broken on `alpine:edge`.

**Comment by neomantra** [comment]
We have released an Alpine 3.13 version.... not sure if people generally want edge...

**Comment by neomantra** [comment]
I'm not going to do releases from `edge` but only specific releases.  It is too unstable otherwise.
